domains/astika.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Personal portfolio website",
+  "repo": "https://github.com/astika327-dev/portofolio",
+  "owner": {
+    "username": "astika327-dev",
+    "email": "astika327@gmail.com"
+  },
+  "record": {
```

records, with **"S"**
```suggestion
  "records": {
```